backend/cmd/headlamp.go

```diff
@@ -771,6 +891,11 @@ func (c *HeadlampConfig) getClusters() []Cluster {
 
 	for _, context := range contexts {
 		context := context
+
+		if context.Internal {
```

Comment:
https://github.com/headlamp-k8s/headlamp/pull/1303#discussion_r1423262134

Continuing this. Since `/config` show's all the clusters saved in the cache, we don't want others to see when someone created a stateless cluster since their proxy is stored in cache.